sbt-plugin-test/crossProjectRef/project/build.sbt

```diff
@@ -0,0 +1,4 @@
+addSbtPlugin("org.scala-js" % "sbt-scalajs" %
+  org.scalajs.core.ir.ScalaJSVersions.current)
+
+libraryDependencies += "org.eclipse.jetty" % "jetty-server" % "9.2.3.v20140905"
```

Comment:
libraryDependencies += "org.eclipse.jetty" % "jetty-server" % "9.2.3.v20140905"

Ok

    addSbtPlugin("org.scala-js" % "sbt-scalajs" % org.scalajs.core.ir.ScalaJSVersions.current)

This one is necessary cause it individual project.